Tests/test_Phylo_CDAO.py

```diff
@@ -68,8 +72,12 @@ def test_write(self):
                 pass
             else:
                 # Can't sort lists with None on Python 3 ...
-                self.assertFalse(None in p1, "Bad input values for %s: %r" % (prop_name, p1))
-                self.assertFalse(None in p2, "Bad output values for %s: %r" % (prop_name, p2))
```

Comment:
This isn't a black issue, but since you are working on this, I think these two lines could be re-written using ``self.assertNotIn`` (new in Python 3.1):

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNotIn

I'll open a general issue on this as part of the Python 2/3 cleanup work.

Reply:
Ok, done.